src/components/studies/participants/ParticipantTableGrid.tsx

```diff
@@ -2,59 +2,87 @@ import { Button, Paper } from '@material-ui/core'
 import { makeStyles } from '@material-ui/core/styles'
 import { ColDef, DataGrid, ValueGetterParams } from '@material-ui/data-grid'
 import React, { FunctionComponent } from 'react'
-import { useUserSessionDataState } from '../../../helpers/AuthContext'
-import ParticipantService from '../../../services/participants.service'
 import {
   ParticipantAccountSummary,
-  StringDictionary
+  StringDictionary,
 } from '../../../types/types'
+import ParticipantTablePagination from './ParticipantTablePagination'
+import { useUserSessionDataState } from '../../../helpers/AuthContext'
+import ParticipantService from '../../../services/participants.service'
 
 const useStyles = makeStyles(theme => ({
   root: {},
+  showEntryText: {
+    fontFamily: 'Lato',
```

Comment:
please import the font and reference that rather than my string. Helps to avoid typos